workloads/virtual-machines/interfaces-and-networks.md

```diff
@@ -18,34 +18,83 @@ All possible configuration options are available in the
 [Interface API Reference](https://kubevirt.io/api-reference/master/definitions.html#_v1_interface)
 and [Network API Reference](https://kubevirt.io/api-reference/master/definitions.html#_v1_network).
 
-> **Note:** Currently, the only supported network type is `pod`.
-
 ## Backend
 
 Network backends are configured in `spec.networks`. A network must have a
 unique name. Additional fields declare which logical or physical device the
 network relates to.
 
+Each network should declare its type by defining on of the following fields:
+
+| Type | Description |
+|--|--|
+| `pod` | Default Kubernetes network |
+| `multus` | Secondary network provided using Multus |
+
+### pod
+
 A `pod` network represents the default pod `eth0` interface configured by
-cluster SDN solution that is present in each pod.
+cluster network solution that is present in each pod.
 
-> **Note:** Currently, the only supported network type is `pod`. In the future,
-> more network types may be supported.
+```yaml
+kind: VM
+spec:
+  domain:
+    devices:
+      interfaces:
+        - name: default
+          bridge: {}
+  networks:
+  - name: default
+    pod: {} # Stock pod network
+```
+
+### multus
+
+It is also possible to connect VMI to a secondary networks using
+[Multus](https://github.com/intel/multus-cni). This assumes multus plugin is
+installed accross your cluster and NetworkAttachmentDefinition CRD was created.
+
+First of all, NetworkAttachmentDefinition for desired secondary network must be
```

Comment:
First, **a** `NetwokrAttachmentDefinition` for **the** desired secondary network must be created.